script.js

```diff
@@ -0,0 +1,97 @@
+import {languageDetailsMap} from './details.js';
+
+document.addEventListener("DOMContentLoaded", function() {
+
+    const RGB_BLACK_COLOR = "rgb(0, 0, 0)";
+    const RGB_YELLOW_COLOR = "rgb(255, 255, 0)";
+
+    const WRAPPER_ELEMENTS = Array.from(document.getElementsByClassName("wrapper"));
+    const PHP_IMAGE = document.getElementById("php");
+
+
+    document.querySelector(".red").addEventListener("click", () => {
+        setDefaultProperties();
+        changeProperties();
+    });
+
+    document.querySelector(".blue").addEventListener("click", () => {
+        setDefaultProperties();
+        setBlurWithoutLastElement(PHP_IMAGE);
+        changeBackgroundColor(PHP_IMAGE, RGB_YELLOW_COLOR)
+        setLanguageDetailsToElementByKey(PHP_IMAGE.id);
+    });
+
+    async function changeProperties() {
+        for (let i = 0; i <= WRAPPER_ELEMENTS.length; i++) {
+            let index = getRandomIndexWithRange(0, WRAPPER_ELEMENTS.length);
+            let element = WRAPPER_ELEMENTS[index];
+
+            changeBackgroundColor(element, RGB_YELLOW_COLOR);
+            await sleepMiliseconds(50);
```

Comment:
I nie zrobiłeś tego intervala :-( Ale ok, jak będziesz w stanie to potem komuś wytłumaczyć (np. na rozmowie kwalifikacyjnej), to ok.